documentation/api/io.strimzi.api.kafka.model.kafka.listener.GenericKafkaListenerConfiguration.adoc

```diff
@@ -22,50 +22,58 @@ listeners:
         secretName: my-secret
         certificate: my-listener-certificate.crt
         key: my-listener-key.key
+      externalTrafficPolicy: Local
+      loadBalancerSourceRanges:
+        - 10.0.0.0/8
+        - 88.208.76.87/32  
 # ...
 ----
 
-When the certificate or key in the `brokerCertChainAndKey` secret is updated, the operator will automatically detect it in the next reconciliation and trigger a rolling update of the Kafka brokers to reload the certificate.
+When the certificate or key in the `brokerCertChainAndKey` secret is updated, the operator automatically detects it in the next reconciliation and triggers a rolling update of the Kafka brokers to reload the certificate.
 
 [id='property-listener-config-traffic-policy-{context}']
-= `externalTrafficPolicy`
+= Avoiding hops to other nodes
 
 The `externalTrafficPolicy` property is used with `loadbalancer` and `nodeport` listeners.
-When exposing Kafka outside of Kubernetes you can choose `Local` or `Cluster`.
+When exposing Kafka outside of Kubernetes, you can choose `Local` or `Cluster`.
 `Local` avoids hops to other nodes and preserves the client IP, whereas `Cluster` does neither.
 The default is `Cluster`.
 
 [id='property-listener-config-source-ranges-{context}']
-= `loadBalancerSourceRanges`
+= Providing CIDR source ranges for a loadbalancer
 
-The `loadBalancerSourceRanges` property is only used with `loadbalancer` listeners.
-When exposing Kafka outside of Kubernetes use source ranges, in addition to labels and annotations, to customize how a service is created.
+The `loadBalancerSourceRanges` property is for `loadbalancer` listeners only. 
+When exposing Kafka outside of Kubernetes, use CIDR (Classless Inter-Domain Routing) source ranges in addition to labels and annotations to customize how a service is created.
 
-.Example source ranges configured for a loadbalancer listener
-[source,yaml,subs=attributes+]
+.Example `loadbalancer` listener configuration to provide source ranges
+[source,yaml,subs="attributes+"]
 ----
 listeners:
   #...
   - name: external3
     port: 9094
     type: loadbalancer
-    tls: false
+    tls: true
+    authentication:
+      type: tls
     configuration:
+      brokerCertChainAndKey:
+        secretName: my-secret
+        certificate: my-listener-certificate.crt
+        key: my-listener-key.key
       externalTrafficPolicy: Local
       loadBalancerSourceRanges:
         - 10.0.0.0/8
-        - 88.208.76.87/32
-      # ...
+        - 88.208.76.87/32  
```

Comment:
I would focus on the core configuration only here -> on the `loadBalancerSourceRanges`. The rest  - authentication and brokerCertChainAndKey - can be IMHO removed.